examples/spark/spark-worker-controller.yaml

```diff
@@ -0,0 +1,24 @@
+kind: ReplicationController
+apiVersion: v1
+metadata:
+  name: spark-worker-controller
+  labels:
+    component: spark-worker
```

Comment:
I actually think it makes sense to have the controller named "foo-controller" but the label named "role: foo" or "component: foo", etc, and I thought I had seen this pattern elsewhere. I'm not sure which is dominant?